.github/pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+## What changes are proposed in this pull request?
+ 
+(Please fill in changes proposed in this fix)
+ 
+## How is this patch tested?
+ 
+(Details)
+ 
+## Release Notes
+ 
+### Is this a user-facing change? If so, please give a 1-2 sentence description for MLflow users.
+ 
+(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)
```

Comment:
If it's not a user-facing change, should the user also add the `rn/none` label to ensure that this field is ignored? If so, maybe we should tell the user to add the `rn/none` label here.

Reply:
Can we make it easy with a 

[ ] This change is not user facing (mark box with an 'x' or click box after saving PR)